contracts/ERC721CM.sol

```diff
@@ -187,14 +194,17 @@ contract ERC721CM is IERC721M, ERC721ACQueryable, Ownable, ReentrancyGuard {
             emit UpdateStage(
                 i,
                 newStages[i].price,
+                newStages[i].mintFee,
                 newStages[i].walletLimit,
                 newStages[i].merkleRoot,
                 newStages[i].maxStageSupply,
                 newStages[i].startTimeUnixSeconds,
```

Comment:
nit pick, but i think splitting into a variable might be more gas efficient here
`MintStageInfo calldata stageInfo = newStages[i];`

Reply:
Not sure if the compiler will optimize. But I think it's ok as we only call setStages once.